### DISCUSSION TITLE
Assistance Required for Benefits Activation After Approval Status in GitHub Education

### DISCUSSION THREAD

**Discussion by Sitapuriya** [community/community#135433] [GitHub Education]
### Hi everyone,

Hello GitHub Education Community,

I recently received approval for the GitHub Education benefits. However, it has been five days since my approval, and I have not yet received access to the benefits. I am reaching out to seek assistance in resolving this issue.

Could someone from the Education Community team please help expedite the activation process or guide me on the necessary steps to ensure that my benefits are activated?

Thank you in advance for your support.

Best regards,
Shubham Pal

**Comment by geniuszly** [comment]
Hi! This problem is observed if, for example, you entered the wrong data in the questionnaire, meaning you did not add the translation of the entire student ID on a separate paper next to the ticket itself. Since the support from github will not be clear exactly what you have written there. If, for example, everything is written there in French, put paper next to it and manually rewrite some values - in particular, Last name, first name, patronymic, when you arrived, date of birth, ticket number, etc. - and all this is in English, respectively. In addition, the verification takes from 3 to 7 days, keep this in mind. Re-examine the requirements for obtaining this package, the availability of 2fa, etc.

**Reply by Sitapuriya** [reply]
everything is correct

**Reply by geniuszly** [reply]
- Make sure you've received an email confirmation of your GitHub Education approval. Sometimes, the benefits activation link or further instructions might be in that email.
- Go to the [GitHub Education page](https://education.github.com/pack) and log in with your GitHub account. This dashboard usually provides details on your status and the benefits available to you. 
- If you're not seeing the benefits, try clearing your browser cache or accessing the dashboard from an incognito window or a different browser.
-  If it's been more than a few days and you still don't have access, you can contact GitHub Support directly. Provide them with your GitHub username, the date of approval, and any relevant information.
-  In rare cases, there might have been an issue with the application process. Double-check your application status on the Education dashboard, and if necessary, you might need to reapply.
